### PR TITLE
Support GHC 8.8

### PR DESCRIPTION
**Pull request by fumieval**
Pardon an automatically-posted pull request; this is a part of [Operation Vanguard](https://github.com/haskell-vanguard/haskell-vanguard). Note that this does not guarantee that tests and benchmarks are buildable.

### CODE REVIEW

File: Data/Primitive/Array.hs [hunk 1]
```diff
@@ -702,8 +702,10 @@ instance Monad Array where
          *> fill (off + lsb) sbs smb
 
 #if !(MIN_VERSION_base(4,13,0))
+#if !MIN_VERSION_base(4,13,0)
```
Comment (chessai) [inline Data/Primitive/Array.hs]: this is redundant

File: Data/Primitive/Array.hs [hunk 2]
```diff
@@ -702,8 +702,10 @@ instance Monad Array where
...
   fail = Fail.fail
 #endif
+#endif
```
Comment (chessai) [inline Data/Primitive/Array.hs]: this is redundant

File: Data/Primitive/SmallArray.hs [hunk 3]
```diff
@@ -810,8 +810,10 @@ instance Monad SmallArray where
        *> fill (off + length sb) sbs smb
 
 #if !(MIN_VERSION_base(4,13,0))
+#if !MIN_VERSION_base(4,13,0)
```
Comment (chessai) [inline Data/Primitive/SmallArray.hs]: this is redundant

**Comment by chessai** [comment]
i'm not sure we'd want to take this in favour of #234, which is already queued up for the 0.7 release. though, in #234 we probably also want to bump the version of base in the benchmarks/test suite

**Comment by fumieval** [comment]
Sorry, my script went in a wrong way; fixed redundant changes. I didn't realise there was already a pull request.

**Comment by cartazio** [comment]
theres already support on hackage 0.6.1.3, and a matching tag on git, 

for managing a package set, master is never going to be the stable interface ... 

as for master, i've a patch that just needs to be rebased ontop of the recent docfix

**Pull request closed without merging**